sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -311,20 +357,33 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
                             "Non-retryable server side error: %s.",
                             response_error.message,
                         )
+                        # Track dropped items in customer statsbeat, non-retryable scenario
+                        if self._customer_statsbeat_metrics and self._should_collect_customer_statsbeat():
+                            _track_dropped_items(self._customer_statsbeat_metrics, envelopes, DropCode.CLIENT_EXCEPTION, response_error)
```

Comment:
This is not a client_exception, use the action status code as the drop code. No need for reason in this case.